advanced_functionality/tensorflow_bring_your_own/tensorflow_bring_your_own.ipynb

```diff
@@ -0,0 +1,814 @@
+      "\u001b[36malgo-1-DZB6N_1  |\u001b[0m 2018-08-01 08:14:23.018543: I tensorflow_serving/servables/tensorflow/saved_model_warmup.cc:83] No warmup data file found at /opt/ml/model/export/Servo/1533111259/assets.extra/tf_serving_warmup_requests\n",
+      "\u001b[36malgo-1-DZB6N_1  |\u001b[0m 2018-08-01 08:14:23.033650: I tensorflow_serving/core/loader_harness.cc:86] Successfully loaded servable version {name: cifar10_model version: 1533111259}\n",
+      "\u001b[36malgo-1-DZB6N_1  |\u001b[0m 2018-08-01 08:14:23.038377: I tensorflow_serving/model_servers/main.cc:323] Running ModelServer at 0.0.0.0:8500 ...\n",
+      "\u001b[36malgo-1-DZB6N_1  |\u001b[0m [warn] getaddrinfo: address family for nodename not supported\n",
+      "\u001b[36malgo-1-DZB6N_1  |\u001b[0m 2018-08-01 08:14:23.041668: I tensorflow_serving/model_servers/main.cc:333] Exporting HTTP/REST API at:localhost:8501 ...\n",
+      "\u001b[36malgo-1-DZB6N_1  |\u001b[0m [evhttp_server.cc : 235] RAW: Entering the event loop ...\n",
+      "!\u001b[36malgo-1-DZB6N_1  |\u001b[0m 172.18.0.1 - - [01/Aug/2018:08:14:23 +0000] \"GET /ping HTTP/1.1\" 200 2 \"-\" \"-\"\n"
+     ]
+    }
+   ],
+   "source": [
+    "from sagemaker.estimator import Estimator\n",
+    "\n",
+    "hyperparameters = {'train-steps': 100}\n",
+    "\n",
+    "instance_type = 'local'\n",
+    "\n",
+    "estimator = Estimator(role=role,\n",
+    "                      train_instance_count=1,\n",
+    "                      train_instance_type=instance_type,\n",
+    "                      image_name='tensorflow_cifar10_example:latest',\n",
+    "                      hyperparameters=hyperparameters)\n",
+    "\n",
+    "estimator.fit('file:///tmp/cifar-10-data')\n",
+    "\n",
+    "predictor = estimator.deploy(1, instance_type)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Making predictions using Python SDK\n",
+    "\n",
+    "In order to do some predictions, we will use an image, which will be converted using OpenCV, into a json format to send as an inference request."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[36malgo-1-DZB6N_1  |\u001b[0m 172.18.0.1 - - [01/Aug/2018:08:14:28 +0000] \"POST /invocations HTTP/1.1\" 200 236 \"-\" \"-\"\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "{'predictions': [{'probabilities': [0.0203898,\n",
+       "    0.00186337,\n",
+       "    0.00125851,\n",
+       "    0.116194,\n",
+       "    0.860205,\n",
+       "    2.24463e-10,\n",
+       "    3.87913e-06,\n",
+       "    8.29227e-05,\n",
+       "    1.4244e-06,\n",
+       "    1.60669e-06],\n",
+       "   'classes': 4}]}"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import cv2\n",
```

Comment:
Make sure to test the notebook in SageMaker Notebooks. Is it possible that they do not have all modules installed.